code/modules/mob/living/simple_animal/friendly/raven.dm

```diff
@@ -0,0 +1,57 @@
+var/cooldown = world.time//DO MOBS NOT HAVE A COOLDOWN VARIBLE!
+/mob/living/simple_animal/pet/raven
+	name = "raven"
```

Comment:
This var needs scoping to the mob level, and possibly naming something more sane.

Reply:
Alright ill throw it into the mob level. I am not sure what to name it, I am probably not qualified enough for sanity yet.